packages/client/src/grpc-retry.ts

```diff
@@ -0,0 +1,86 @@
+import {
+  InterceptingCall,
+  Interceptor,
+  ListenerBuilder,
+  Metadata,
+  RequesterBuilder,
+  StatusObject,
+} from '@grpc/grpc-js';
+import * as grpc from '@grpc/grpc-js';
+
+const maxRetries = 10;
+const retryableCodes = [
```

Comment:
It's probably far from being an issue but this could be a `Set` for "faster" lookup.